scripts/valhalla_build_timezones

```diff
@@ -1,18 +1,16 @@
 #!/bin/sh
 
 error_exit() {
-  echo "$1" 1>&2
-  #exit 1
+  echo "error: $1" 1>&2
+  exit 1
```

Comment:
this will allow users with geos 3.9 to not error out from double free

```suggestion
  exit $(pkg-config geos --modversion | grep -cvF "3.9")
```

Reply:
Didn't get the problem. How geos affects exiting with failure if some step fails?

Reply:
yeah spatialite which is what we use here, links libgeos, certain versions 3.9.* seem to have a bug where they double free. this causes spatialite to crash which causes this script to fail. i commented out the exit because of this.